src/scripts/components/importExport/Import.js

```diff
@@ -424,7 +424,16 @@ export default class Import extends MnObject {
             const data      = JSON.parse(res);
 
             if (path[2] === 'notes') {
-                return this.importOldData('notes', data);
+                return this.importOldData('notes', data).then(() => {
+                    let options = { zip, data, name: file.name };
+
+                    this.readMarkdown(options).then(() => {
+                        // console.log('import: readLegacyFile(): readMarkdown: ');
+                        // console.log(data);
```

Comment:
Could you erase those comments or use them with log() please ?

Reply:
@Maks-s I changed them for log()